### PR TITLE
JBPM-7161 - Fix new case project configuration

### PR DESCRIPTION
**Pull request by tsurdilo**
@mswiderski this puts back the use of configureCaseProjectCallback which was removed in commit 
https://github.com/kiegroup/jbpm-wb/commit/57c1fd7350f4f7f06c6037aeb3f4dd0f9e7102d7#diff-f7a96202b473a4c949f4d3fd20642d1d
and also fixes the path issues in creating the "/.caseproject" dot file

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@tsurdilo wondering if these changes should be covered in the related test classes?

**Comment by tsurdilo** [comment]
@cristianonicolai from what i seen yes, but if you see anything that could be added please let me know.

**Comment by cristianonicolai** [comment]
@tsurdilo perhaps make sure that _configureCaseProjectCallback_ is used in the _init_/_wizard.start_ methods. Also a test to ensure the remaining part of _configureNewCaseProject_ works as expected when an exception is thrown in the new try block ( _ioService.write_? ).

File: jbpm-wb-integration/jbpm-wb-integration-backend/src/main/java/org/jbpm/workbench/wi/backend/server/casemgmt/service/CaseProjectServiceImpl.java [hunk 1]
```diff
@@ -83,12 +83,16 @@ public void configureNewCaseProject(WorkspaceProject project) {
...
         // add empty .caseproject file as marker
-        String rootPathString = project.getRootPath().toURI().toString() + separator + CASE_PROJECT_DOT_FILE;
-        Path rootPath = ioService.get(URI.create(rootPathString));
-        ioService.write(rootPath,
-                        "");
-        logger.debug("Added caseproject marker (dot) file at {}",
-                     rootPath);
+        try {
+            String caseProjectDotFilePath = project.getMainModule().getRootPath().toURI().toString() + CASE_PROJECT_DOT_FILE;
+            ioService.write(ioService.get(new URI(caseProjectDotFilePath)),
```
Comment (cristianonicolai) [inline jbpm-wb-integration/jbpm-wb-integration-backend/src/main/java/org/jbpm/workbench/wi/backend/server/casemgmt/service/CaseProjectServiceImpl.java]: @tsurdilo perhaps test if marker file already exists, similar to what is done for the deployment descriptor file?

**Pull request merged**